packages/checked_oint/checked_oint.0.2.0/opam

```diff
@@ -17,17 +17,7 @@ depends: [
   "alcotest" {>= "1.7.0"}
   "odoc" {with-doc}
 ]
-available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32"
-x-ci-accept-failures: [
-  "fedora-38"
-  "fedora-39"
-  "fedora-40"
-  "opensuse-15.5"
-  "opensuse-tumbleweed"
-  "oraclelinux-8"
-  "oraclelinux-9"
-  "freebsd"
-]
+available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32" & os != "freebsd" & os-distribution != "fedora" & os-distribution != "oraclelinux" & os-family != "opensuse" & switch != "5.3~alpha1"
```

Comment:
You need this
```suggestion
available: arch != "arm32" & arch != "x86_32" & arch != "ppc64" & os != "win32" & os != "freebsd" & os-distribution != "fedora" & os-distribution != "oraclelinux" & os-family != "opensuse" & os-family != "suse"
```

if the library is not compatible with ocaml 5.3 I would suggest an upper bound on the ocaml dependency instead. This is the alpha release of the compiler though, so I thunk it is better not to add any extra compiler constraint for the moment

Reply:
I forgot to mention that the various os flags are documented here https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values